clang/tools/clang-scan-deps/ClangScanDeps.cpp

```diff
@@ -792,10 +792,15 @@ int clang_scan_deps_main(int argc, char **argv, const llvm::ToolContext &) {
 
   llvm::cl::PrintOptionValues();
 
+  // Expand response files in advance, so that we can "see" all the arguments
+  // when adjusting below.
+  auto ResponseExpander = expandResponseFiles(std::move(Compilations),
```

Comment:
"Expander" sounds like a function object that you need to invoke to expand the compilations, while the result of `expandResponseFiles()` behaves like something that's already been expanded. I don't think the fact that this has been expanded is really relevant to the rest of the code here.

```suggestion
  Compilations = expandResponseFiles(std::move(Compilations),
```

Reply:
As suggested.